doc/source/data/monitoring-your-workload.rst

```diff
@@ -0,0 +1,90 @@
+.. _monitoring-your-workload:
+
+Monitoring Your Workload
+========================
+
+This section helps you debug and monitor the execution of your :class:`~ray.data.Dataset` by viewing the:
+
+* Ray Data overview :ref:`Ray Data Overview`
+* Ray dashboard metrics :ref:`Ray Dashboard Metrics`
+* Ray Data logs :ref:`Ray Data Logs`
+
+Ray Data Dashboard
+------------------
+
+Ray Data emits Prometheus metrics in real-time while a Dataset is executing. These metrics are tagged by both dataset and operator, and are displayed in multiple views across the Ray dashboard.
+
+.. note::
+   Most metrics are only available for physical operators that use the map operation.
```

Comment:
can we add examples of which Dataset APIs this includes? `ds.map`, `ds.map_batches`, `ds.flat_map`, etc.